.prettierignore

```diff
@@ -61,7 +61,7 @@ docker-compose.override.yml
 /app/javascript/mastodon/features/emoji/emoji_map.json
 
 # Ignore locale files
-/app/javascript/mastodon/locales
+/app/javascript/mastodon/locales/data/*.json
```

Comment:
It seems like the JSON files don't need to be ignored anymore after the migration from `translationRunner.js`

Reply:
They are automatically generated, so we dont want Prettier to process them